website/content/api-docs/secret/pki.mdx

```diff
@@ -1908,6 +1908,82 @@ $ curl \
 }
 ```
 
+### List certificates with details
+
+This endpoint returns a list of the current certificates by serial number and provides details such as CN, issuer, key_type, expiration, not_before and upto 5 DNS names.
```

Comment:
nit:

```suggestion
This endpoint returns a list of the current certificates by serial number and provides details such as CN, issuer, key_type, expiration, not_before and up to 5 DNS names.
```

What do you think of collapsing this documentation into the `LIST /pki/certs` version instead? Some of the info is redundant and you could say detailed information is only present on this endpoint instead.

I think you can see it with things like import: https://openbao.org/api-docs/secret/pki/#import-ca-certificates-and-keys 

Otherwise, the page gets rather long... 

Thoughts?